Modules/Registration/Common/wrapping/itkPointSetToPointSetMetric.wrap

```diff
@@ -7,4 +7,9 @@ itk_wrap_class("itk::PointSetToPointSetMetric" POINTER_WITH_2_SUPERCLASSES)
       itk_wrap_template("PS${ITKM_${t}}${d}" "itk::PointSet< ${ITKT_${t}},${d} >, itk::PointSet< ${ITKT_${t}},${d} >")
     endforeach()
   endforeach()
-itk_end_wrap_class()
+  if(d3 AND d2)
+    foreach(t ${types})
+      itk_wrap_template("PS${ITKM_${t}}23" "itk::PointSet< ${ITKT_${t}},2 >, itk::PointSet< ${ITKT_${t}},3 >")
```

Comment:
Should 3D->2D case be wrapped too? This does it for 2D->3D.

Reply:
I believe only 2D->3D can be solved with a pair of images.  There is no way to solve 3D->2D.    That is, the fixed point set must be 2D, and the perspective transform moves the 3D points into 2D space and the metric is a 2D to 2D metric.   Determining how to project 2D points into 3D space requires multiple projection images.